### DISCUSSION TITLE
Removing CT/Meter/RS485 connector from inverter

### DISCUSSION THREAD

**Discussion by LevittS** [nathanmarlor/foxess_modbus#338] [Q&A] [answered]
Hi,

I'm probably being stupid but I've been unable to work out how to remove the CT/Meter/RS485 connector from my H1 inverter.

Is it secured by a clip, locking nut, or similar?

I've tried to unscrew what possibly could be a locking nut, but it feels stupidly tight if it is, and as my loft is low causing the inverter to be mounted quite low and with the lighting being far from ideal I can't get to somewhere where I can clearly see what's going on.

I've had a bit of a search around and can't find anything describing a process, so I assume its quite simple, but mainly for the fear of breaking something hoped to get some hints if I could.

Thanks in advance

**Comment by FozzieUK** [comment, marked as answer]
It’s clipped in, there are 2 serated wings that stick out slightly near the top - squeeze them in and pull down on the connector

**Reply by FozzieUK** [reply]
It looks like this when disconnected.

![IMG_7906](https://github.com/nathanmarlor/foxess_modbus/assets/113460294/8eb2d6cc-5599-4ece-a4e6-761b238511e0)

**Comment by LevittS** [comment]
Exactly what I was after - thanks.
